src/qt/bitcoingui.cpp

```diff
@@ -318,8 +318,8 @@ void BitcoinGUI::createActions()
     verifyMessageAction = new QAction(tr("&Verify message..."), this);
     verifyMessageAction->setStatusTip(tr("Verify messages to ensure they were signed with specified Bitcoin addresses"));
 
-    openRPCConsoleAction = new QAction(tr("&Debug window"), this);
-    openRPCConsoleAction->setStatusTip(tr("Open debugging and diagnostic console"));
+    openRPCConsoleAction = new QAction(tr("&Node window"), this);
```

Comment:
This needs to be updated:
https://github.com/bitcoin/bitcoin/blob/69a6f1ad1f7ca52b2524ab8322bfa89a9b0ee61b/src/qt/bitcoingui.cpp#L415

But I'm not sure about this shortcut, there's now a shortcut for each tab so maybe drop these?

Reply:
Also `&N` mnemonic is associated with "Network Traffic" which is a tab in the node window.

Reply:
> This needs to be updated:
> https://github.com/bitcoin/bitcoin/blob/69a6f1ad1f7ca52b2524ab8322bfa89a9b0ee61b/src/qt/bitcoingui.cpp#L415
> 
> But I'm not sure about this shortcut, there's now a shortcut for each tab so maybe drop these?

So taking out the entire line since [these](https://github.com/bitcoin/bitcoin/commit/091747b46ecf06244ce2650028f1833b2e7c5062) [commits](https://github.com/bitcoin/bitcoin/commit/8019b6b150ff7444195a238470414c9deec5bf74) make the shortcut unnecessary?

Reply:
> Also `&N` mnemonic is associated with "Network Traffic" which is a tab in the node window.

Do you have a suggestion as to what mnemonic to replace it with? Or would you rather it be removed completely:

```cpp
openRPCConsoleAction = new QAction(tr("Node window"), this);
```

Reply:
I'd drop it, not sure about others though.

Reply:
Got it :+1: